javascript/javascript1/week4/script.js

```diff
@@ -0,0 +1,97 @@
+// voice assistant 
+
+let userName= "";
+const todolist=[];
+
+function getReply(command){
+if (command==="Hello my name is Benjamin" && userName==null){
+    return `You are not a person`
+}
+    if (command ==="Hello my name is Benjamin" && userName===""){
```

Comment:
You should check which name is written ie it should also work with `getReply("Hello my name is Hanna");`